python/tk_multi_launchapp/software_entity_launcher.py

```diff
@@ -254,6 +261,7 @@ def _get_sg_software_entities(self):
             "image",
             "sg_engine",
             "sg_versions",
+            "sg_products",
```

Comment:
Yep. And the filters specified for the `shotgun.find()` call when searching for available Software entities will also need to be updated.  Thankfully, they will actually be simplified a bunch 👍